README.md

```diff
@@ -59,13 +59,12 @@ tool, but we don't offer documentation or support in doing so.  Adapting one of
+| if connecting to Microsoft 365    | [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)                  | 2.29+   | `az --version`   | [Cloud Application Administrator](https://learn.microsoft.com/en-us/azure/active-directory/roles/permissions-reference#cloud-application-administrator)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
I'd include or at least mention `Privileged Role Administrator` to be consistent with [MSFT 365 sources docs](https://github.com/Worklytics/psoxy/blob/main/docs/sources/msft-365/readme.md#required-roles). It is true that role is not required to deploy the examples and create the applications, but it will be required to setup them after with admin consent

Reply:
thanks; I'm split this more clearly into data source instructions on another branch